tests/test_app/test_dist.py

```diff
@@ -213,7 +216,6 @@ def test_composes_with_write_tabular(self):
             dist_calc = dist_app.fast_slow_dist(distance="hamming", moltype="protein")
             _ = dist_calc + writer
```

Comment:
Do we always need have an assignment for result of composable apps? or 
Can we write this expression without assignment like this: `app1+app2` without assigning result to a variable? I imagine `app1+app2` like calling two functions: `function1(function2())`

Reply:
I prefer to do the assignment but it's possible to do what you've said